src/Sylius/Bundle/AttributeBundle/Form/Type/AttributeType/Configuration/OptionListAttributeConfigurationType.php

```diff
@@ -0,0 +1,83 @@
+<?php
+/**
+ * @author    Fernando Caraballo Ortiz <fernando.ortiz@reiss.com>
+ * @date      07/11/2016
+ * @copyright Copyright (c) Reiss Clothing Ltd.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\AttributeBundle\Form\Type\AttributeType\Configuration;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\FormEvent;
+use Symfony\Component\Form\FormEvents;
+
+/**
+ * @author Fernando Caraballo Ortiz <fernando.ortiz@reiss.com>
+ */
+class OptionListAttributeConfigurationType extends AbstractType
+{
+    const FORMAT_DROPDOWN           = 'dropdown';
+    const FORMAT_MULTIPLE_SELECTION = 'multiple_selection';
+
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('format', 'choice', [
+                'choices' => [
+                    self::FORMAT_DROPDOWN           => ucfirst(self::FORMAT_DROPDOWN),
```

Comment:
definitely use them on the value side, but for labels it's a text string - why not just write it?